### PR TITLE
OSDOCS-8302 4.12.40 z-stream release notes

### PR DESCRIPTION
**Pull request by cbippley**
Version(s):
4.12

Issue:
[OSDOCS-8302](https://issues.redhat.com/browse/OSDOCS-8302)

Link to docs preview:

QE review:
- [ ] QE has approved this change.
not needed

Additional information:
Expected merge date 10/25/23. Links will not work until then.

### CODE REVIEW

**Review by skrthomas — COMMENTED** [review]
/lgtm

File: release_notes/ocp-4-12-release-notes.adoc [hunk 1]
```diff
@@ -4207,4 +4207,23 @@ $ oc adm release info 4.12.39 --pullspecs
...
 ==== Updating
 
+To update an existing {product-title} 4.12 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
+
+[id="ocp-4-12-40"]
+=== RHSA-2023:5896 - {product-title} 4.12.40 bug fix and security update
+
+Issued: 2023-10-25
+
+{product-title} release 4.12.40, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2023:5896[RHSA-2023:5896] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2023:5898[RHBA-2023:5898] advisory.
```
Comment (jeana-redhat) [inline release_notes/ocp-4-12-release-notes.adoc:4217]: https://access.redhat.com/errata/RHSA-2023:5896 is good but I get a 404 for https://access.redhat.com/errata/RHBA-2023:5898

**Pull request merged**